source/security/authentication.txt

```diff
@@ -0,0 +1,531 @@
+.. _pymongo-auth:
+
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: authorize, boto, ecs, aws
+
+Overview
+--------
+
+This guide describes the mechanisms you can use in {+driver-short+} to authenticate
+users.
+
+.. important:: Percent-Escaping
```

Comment:
S: Unless you're trying to correlate the phrases "percent-escaping" and "percent-encoding"
```suggestion
.. important:: Percent-Encoding
```

Reply:
yep, good catch